ui/package.json

```diff
@@ -15,6 +15,7 @@
     "test:debug": "react-app-rewired --inspect-brk test --watchAll=false --run-in-band",
     "test": "react-app-rewired test --watchAll=false",
     "lint": "yarn eslint src --ext .ts,.tsx",
+    "libyear": "libyear",
```

Comment:
libyear can be invoked with `yarn run libyear` with or without this line. However, I think this line is useful to document the command and indicate that we use `libyear` as a command line tool.